roles/openshift_control_plane/tasks/main.yml

```diff
@@ -280,22 +283,19 @@
   - debug:
       msg: "{{ logs_node.stdout_lines }}"
   - name: Get API logs
-    command: >
-      /usr/local/bin/master-logs api api
+    command: journalctl --no-pager -n 300 -u {{ openshift_service_type }}-master-api
```

Comment:
I don't think `journalctl` would work in this case, lets leave it as is now

Reply:
You mean, journal won't have any entries by the time it reaches here? Because, I tested this locally and if restart any of the services, they have journal entries.